packages/testdata/marshaler/packages.json

```diff
@@ -34,6 +34,53 @@
     "/package/multiversion/1.1.0/img/icon.svg"
    ]
   },
+  {
+   "name": "agent_privileges",
+   "title": "Agent Privileges",
+   "version": "1.0.0",
+   "release": "beta",
+   "description": "Test package-specified agent privileges",
+   "type": "solution",
+   "download": "/epr/agent_privileges/agent_privileges-1.0.0.zip",
+   "path": "/package/agent_privileges/1.0.0",
+   "conditions": {
+    "kibana": {
+     "version": "\u003e=7.16.0"
```

Comment:
Umm, this looks weird. But not new in any case.

Reply:
This seems an issue with these tests, registry API responses don't do this encoding.